Lib/unittest/case.py

```diff
@@ -1216,20 +1216,38 @@ def assertCountEqual(self, first, second, msg=None):
             self.fail(msg)
 
     def assertMultiLineEqual(self, first, second, msg=None):
-        """Assert that two multi-line strings are equal."""
-        self.assertIsInstance(first, str, 'First argument is not a string')
-        self.assertIsInstance(second, str, 'Second argument is not a string')
+        """Assert that two multi-line strings are equal.
+         If the assertion fails, then provide an error message with a detailed
+         diff output
```

Comment:
I don't think we really need this expansion of this docstring. The existing single-line docstring already clearly summarizes the purpose of the method. It's assumed that all `assert*` methods attempt to provide a useful message in case of assertion failure.

If we do keep this change for whatever reason, there should be a blank line after the initial one-line summary, and we should include a period at the end of the sentence:

```suggestion
        """Assert that two multi-line strings are equal.

         If the assertion fails, provide an error message with a detailed
         diff output.
```

Reply:
I have reverted back to the single line doc string.